src/phoenix-adapter.js

```diff
@@ -168,11 +168,10 @@ export default class PhoenixAdapter {
 
     message.source = source;
 
-    //TODO: Handle frozen
-    if (this.frozen) {
+    if (this.frozen && (message.dataType === "um" || message.dataType === "u")) {
```

Comment:
We were previously storing away messages that were not `u` or `um` (like the pen drawings) but acting as if they were update messages... This likely caused some of the odd behavior we saw with drawings. The new networking system also runs over one of these custom message types and currently just ignores freeze state